src/features/hud/components/Logout.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useContext } from "react";
+
+import * as Auth from "features/auth/lib/Provider";
+
+import { Modal } from "react-bootstrap";
+import { Button } from "components/ui/Button";
+import { Panel } from "components/ui/Panel";
+
+import close from "assets/icons/close.png";
+
+interface Props {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+export const Logout: React.FC<Props> = ({ isOpen, onClose }) => {
+  const { authService } = useContext(Auth.Context);
+
+  // {Todo: Modify gameState and authMachine for Logout event}
+
+  const handleLogoutConf = (event: React.SyntheticEvent) => {
+    event.preventDefault();
+    onClose();
+    authService.send("ACCOUNT_CHANGED"); // hack used to avoid redundancy
+  };
+
+  return (
+    <Modal show={isOpen} onHide={onClose} centered>
```

Comment:
The padding of the container looks a little bit crowded. I would kindly suggest to add some padding with something like this:

```
      <Panel>
        <div className="p-2">
          <img
            src={close}
            className="h-6 top-6 right-6 absolute cursor-pointer"
            alt="Close Logout Confirmation Modal"
            onClick={onClose}
          />
          <div className="p-4 pb-2 text-center">
            <span className="text-sm">
              Are you sure that you want to &quot;logout&quot;?
            </span>
            <form
              onSubmit={handleLogoutConf}
              className="row justify-between mt-3"
            >
              <Button className="col w-1/3 p-1" type="submit">
                Yes
              </Button>
              <Button
                className="col w-1/3 p-1"
                onClick={(evt) => {
                  evt.preventDefault();
                  onClose();
                }}
              >
                No
              </Button>
            </form>
          </div>
        </div>
      </Panel>
```

so it looks like this:

<img width="592" alt="image" src="https://user-images.githubusercontent.com/160117/162577276-d097f3cb-5dad-483b-babe-845658607a34.png">

Reply:
Thanks, for the UI suggestions. And, welcome to the SFL GitHub community. :)
I had no time to check UI aspects at the time opening this PR.
Will add the classes :)

PS: One suggestion from side, you don't need to paste the whole code in discussions for UI improvements. 👍

Reply:
Thanks Pranav!

> PS: One suggestion from side, you don't need to paste the whole code in discussions for UI improvements. 👍
yeah, I just thought it would be easier than commenting line-by-line